mercury-wallet/umbrel-app.yml

```diff
@@ -0,0 +1,28 @@
+manifestVersion: 1
+id: mercury-wallet
+category: Wallets
+name: Mercury Wallet
+version: "0.8.6"
+tagline: Statechain wallet with coinswap features
+description: >-
+  Mercury is a new Bitcoin layer-2 scaling technology, based on the concept of statechains, 
+  that enables private keys for BTC deposits (UTXOs) to be transfered securely between 
+  owners without requiring an on-chain transaction.
+  This enables users to transfer custody of coins to anyone almost instantly, with increased privacy, and without transaction fees.
+releaseNotes: >-
+  - Swap registration synchronisation
+  - Timeout parameter updates
+  - Receive key verification
+developer: LayerTwoTech
+website: https://mercurywallet.com/
+dependencies:
+repo: https://github.com/layer2tech/mercury-wallet
+support: https://github.com/layer2tech/mercury-wallet/discussions
+port: 3000
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""

```

Comment:
Please could you add submission metadata underneath? Here is an example: https://github.com/getumbrel/umbrel-apps/blob/c009e1505c3e9931e748ceaadb063e5411230e47/agora/umbrel-app.yml#L28-L29

The `submission` value will be the url of this PR.